sdk/storage/Azure.Storage.Queues/tests/QueueClientTests.cs

```diff
@@ -1977,5 +1977,49 @@ public void CanMockClientConstructors()
             mock = new Mock<QueueClient>(new Uri("https://test/test"), new AzureSasCredential("foo"), new QueueClientOptions()).Object;
             mock = new Mock<QueueClient>(new Uri("https://test/test"), mockTokenCredential, new QueueClientOptions()).Object;
         }
+
+        [RecordedTest]
+        [TestCase("")]
+        [TestCase("ra")]
+        [TestCase("raup")]
+        [TestCase("paru")]
+        [TestCase("zaxpqr")]
+        public void QueueClientPolicyPermissions_checkIfSetCorrectly(string permissions)
```

Comment:
Change this to
`public async Task QueueClientPolicyPermissions_checkIfSetCorrectlyAsync(string permissions)`

Reply:
Actually nevermind ignore me